typespec-extension/src/main/java/com/azure/autorest/TypeSpecPlugin.java

```diff
@@ -260,7 +260,6 @@ public TypeSpecPlugin(EmitterOptions options, boolean sdkIntegration) {
             SETTINGS_MAP.put("license-header", "SMALL_TYPESPEC");
 
             SETTINGS_MAP.put("sync-methods", "sync-only");
-            SETTINGS_MAP.put("stream-style-serialization", true);
```

Comment:
default is here
https://github.com/Azure/autorest.java/blob/a118c84ba4c6f6b80fea925c2d664911abbc5ad3/typespec-extension/src/main/java/com/azure/typespec/model/EmitterOptions.java#L46-L47